packages/babel-preset/CHANGELOG.md

```diff
@@ -7,6 +7,14 @@ and adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 <!-- ## Unreleased -->
 
+### Added
+
+- Created a `common` babel-preset [[#231](https://github.com/Shopify/web-foundation/pull/231)]
```

Comment:
Do we want to make a bigger deal out of this being experimental for now?

I'm assuming this is a minor release for now, and in a few weeks once it proves itself we will cut a major version that exposes the common preset in `index.js` and remove the web/node presets.